From a0a1f9ddb429b513bd9f946c132bcb6b3ee395a3 Mon Sep 17 00:00:00 2001
From: Zhao Wei Liew <zhaoweiliew@gmail.com>
Date: Sat, 4 Feb 2017 16:12:42 +0800
Subject: [PATCH 078/249] cm: Remove ahbottomnavigation library

We aren't currently using this. The only reason why this was
being synced, was because it's a dependency of the rebased
Gallery app.

Since the Gallery rebase is currently stalled, remove the library
for now.

This essentially reverts commit 58a7613899d1e4aeec65178a9f0da19ef181f475.

Change-Id: I029979484ec3c695aebbe90eb2a32fc23aaa24f1
---
 snippets/cm.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/snippets/cm.xml b/snippets/cm.xml
index 04bcb3a..160f480 100644
--- a/snippets/cm.xml
+++ b/snippets/cm.xml
@@ -3,7 +3,6 @@
 
   <!-- LineageOS additions -->
   <project path="android" name="LineageOS/android" />
-  <project path="external/ahbottomnavigation" name="LineageOS/android_external_ahbottomnavigation" />
   <project path="external/android-visualizer" name="LineageOS/android_external_android-visualizer" />
   <project path="external/bash" name="LineageOS/android_external_bash" />
   <project path="external/cmsdk-api-coverage" name="LineageOS/android_external_cmsdk-api-coverage" />
-- 
2.17.1

